### PR TITLE
Cannot invite non nda user to access group

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

File: pages/edit_group.py [hunk 1]
```diff
@@ -135,12 +173,20 @@ def name(self):
...
 
             def invite_new_member(self, mozillian):
                 search = self.find_element(*self._invite_search_locator)
                 search.send_keys(mozillian)
-                self.wait.until(expected.visibility_of_element_located((
-                    By.XPATH, '//li[contains(text(), "{0}")]'.format(mozillian)))).click()
+                try:
+                    self.wait.until(expected.visibility_of_element_located((
+                        By.XPATH, '//li[contains(text(), "{0}")]'.format(mozillian)))).click()
+                except Exception:
```
Comment (davehunt) [inline pages/edit_group.py]: It's not a good idea to use bare Exception catching. If the element not being visibile, and not clicking on it is an acceptable path then I think we need a separate method for this scenario. At best, this path will incur a delay while we wait for the element to be displayed. We may also miss failures for tests that are expecting this to perform the click.

File: tests/test_group.py [hunk 2]
```diff
@@ -102,3 +102,27 @@ def test_github_non_nda_user_cannot_create_access_group(self, base_url, selenium
...
+        invite_form = group.invitations.invite
+        invites = len(invite_form.invites_list)
+        invite_form.invite_new_member(github_non_nda_user['username'])
+        assert len(invite_form.invites_list) == invites
+
+        # Try to add a non nda user as curator of the access group
+        curators_form = group.access.curators_form
+        curators = len(curators_form.curators_list)
+        curators_form.add_curator(github_non_nda_user['username'])
+        assert len(curators_form.curators_list) == curators
```
Comment (davehunt) [inline tests/test_group.py]: Should invite/add curator be separate tests?

File: variables.json [hunk 3]
```diff
@@ -22,6 +22,11 @@
         "username": "",
         "password": "",
         "secret": ""
+      },
+      "ldap_user": {
```
Comment (davehunt) [inline variables.json]: Do we have a test LDAP account for this with heaviliy restricted access? I'm concerned about storing LDAP credentials for test use. Is it possible to create a test user on staging for these tests? Perhaps @johngian has some thoughts on this?

Comment (viorelaioia) [reply variables.json]: hey @davehunt ! I already have an LDAP test account from Infosec, I will use that one (and share it with you). I also have a mozillians account created in staging for the LDAP test account.

File: tests/test_group.py (hunk 2 above)
Comment (viorelaioia) [reply tests/test_group.py]: I will split the test in 2 separate tests.

**Comment by viorelaioia** [comment]
Adhoc run: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/mozillians.adhoc/266/console

**Comment by stephendonner** [comment]
Gentle ping, @viorelaioia - any plans to rebase/resurrect this PR, or can it be closed, perhaps?  Thanks!

**Comment by viorelaioia** [comment]
hi @stephendonner ! Thanks for the ping, I totally forgot about this PR. I don't plan on working on this soon, so I'm gonna close it for now. I will probably open a new PR with the comments addressed later on.

**Pull request closed without merging**